app.rb

```diff
@@ -0,0 +1,59 @@
+require "sinatra"
+require "active_record"
+require "pry"
+
+require_relative "models/apartment"
+require_relative "models/tenant"
+require_relative "db/connection"
+
+get '/' do
+  "Home page"
+  erb :index
+end
```

Comment:
This could also simply redirect users to an existing view. You also don't need the "Home Page"